codec-http/src/main/java/io/netty/handler/codec/http/DefaultHttpHeaders.java

```diff
@@ -42,6 +43,13 @@
  * Default implementation of {@link HttpHeaders}.
  */
 public class DefaultHttpHeaders extends HttpHeaders {
+    private static final BiPredicate<CharSequence, CharSequence> CASE_INSENSITIVE_CONTAINS =
```

Comment:
This is bimorphic on purpose, but clearly...it can become problematic if the `containsAny` method become too big to be inlined - and the predicate call site bloated of types.
But to be fair it shouldn't happen: this method here is used mostly with a single lambda type.